### ISSUE TITLE
Different output in INT8 and FP16 quantization

### ISSUE THREAD

**Issue by savankoradiya** [ultralytics/yolov5#4990]
Hi @glenn-jocher,
I have converted yolov5s model in INT8 and FP16. I am getting multiple bounding boxes in INT8 Model.

python export.py --weights yolov5s.pt --include saved_model pb tflite

FP16 Output
python detect.py --weights ./yolov5s-fp16.tflite
![yolov5s_fp16_output](https://user-images.githubusercontent.com/43041904/135212346-c39de22c-1a72-4439-b45f-d8fa459d7a7f.jpg)

INT8 Output
python detect.py --weights ./yolov5s-int8.tflite
![yolov5s_int8_output](https://user-images.githubusercontent.com/43041904/135212270-aa4c4d99-b247-4419-8099-bc9af92faca2.jpg)

OS Ubuntu 18.04
Tensorflow 2.4.1
Pytorch 1.9.0

**Comment by SashaAlderson** [comment]
INT8 model should lose some accuracy due to quantization, there is no bug.

**Comment by glenn-jocher** [comment]
@savankoradiya yes, as @SashaAlderson mentioned, you have two very different quantizations which **should** produce different results.

That would nice if we could quantize all the bits away to nothing and still keep the same result but that's not how the world works unfortunately.

**Comment by savankoradiya** [comment]
I understand that we will lose some MAP in int8 quantization and the model will not detect all objects similar to fp16 model.
Here the issue is getting multiple boxes in the same location.
Are there any suggestions to solve that issue?
